dart/lib/src/platform/_web_platform.dart

```diff
@@ -0,0 +1,47 @@
+import 'dart:html' as html;
+import 'platform.dart';
+
+const Platform instance = WebPlatform();
+
+/// [Platform] implementation that delegates to `dart:html`.
+class WebPlatform extends Platform {
+  /// Creates a new [Platform].
+  const WebPlatform();
+
+  @override
+  String get operatingSystem => _browserPlatform();
+
+  @override
+  String get operatingSystemVersion => 'unknown';
```

Comment:
no way of getting it? :(

Reply:
https://stackoverflow.com/questions/9514179/how-to-find-the-operating-system-version-using-javascript

There may be a way :D

I wonder what the Sentry Javascript SDK does.

Reply:
I think it would be better to do this in another PR because it seems like a lot of trial and error to parse the OS version correctly from the browser. Especially the testing of it.

Reply:
So far `operatingSystemVersion` isn't used.

Reply:
no prob, this could be further improvements for Web/Desktop support